src/index.js

```diff
@@ -1,17 +1,61 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import Map from './components/Map'
+
+class Home extends React.Component {
```

Comment:
I think it'd be a good idea to rewrite this in functional component form using Typescript, to match the style we use in our [React project template](https://github.com/azavea/react-redux-typescript-template). There's nothing inherently wrong with doing it this way, but the component patterns are pretty significantly different, so your learning from this breakable toy will be more readily transferable to live projects if you base it off of our team's template. (And personally, after having worked with function-based components and class-based components in GFW, I do prefer the functional components).